lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -12,30 +12,71 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import 'dart:convert';
+import 'dart:io';
+
 import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
 import 'package:meta/meta.dart';
-import 'package:over_react_codemod/src/constants.dart';
 import 'package:over_react_codemod/src/react16_suggestors/react16_utilities.dart';
 import 'package:over_react_codemod/src/util.dart';
 import 'package:source_span/source_span.dart';
 
+import 'boilerplate_constants.dart';
+
 typedef YieldPatch = void Function(
     int startingOffset, int endingOffset, String replacement);
 
-SemverHelper semverHelper;
+/// Returns a [SemverHelper] using the file at [path].
+///
+/// If the file at [path] does not exist, the returned [SemverHelper] assumes
+/// all classes passed to [getPublicExportLocations] are public
+/// (see: [SemverHelper.alwaysPublic] constructor).
+///
+/// If [shouldTreatAllComponentsAsPrivate] is true, the returned [SemverHelper]
+/// assumes all classes passed to [getPublicExportLocations] are private
+/// (see: [SemverHelper.alwaysPrivate] constructor).
+Future<SemverHelper> getSemverHelper(String path,
+    {bool shouldTreatAllComponentsAsPrivate = false}) async {
+  if (shouldTreatAllComponentsAsPrivate) {
+    return SemverHelper.alwaysPrivate();
+  } else {
+    final file = File(path);
+
+    if (await file.exists()) {
```

Comment:
FYI... you don't have to mess with futures if you use the "sync" version... 

```suggestion
    if (file.existsSync()) {
```

Reply:
Nice!